src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -119,33 +119,50 @@ void copy_with_offset(const ov::Tensor& orig, const int32_t offset, ov::Tensor&
     std::copy(orig_data, orig_data + orig.get_size(), padded_data + offset);
 }
 
-ov::AnyMap extract_config_or_default(const ov::AnyMap& config, const std::string& config_name) {
-    ov::AnyMap stage_cfg;
-    if (auto it = config.find(config_name); it != config.end()) {
-        stage_cfg = it->second.as<ov::AnyMap>();
-    } else if (config_name == "PREFILL_CONFIG") {
-        std::map<std::string, std::string> prefill_config = {
-			{ "NPU_USE_NPUW", "YES" },
-			{ "NPUW_FOLD", "YES" },
-			{ "NPUW_DCOFF_TYPE", "f16" },
-			{ "NPUW_DCOFF_SCALE",  "YES" },
-			{ "NPUW_ONLINE_AVOID", "P:RMSNorm/NPU" }
-        };
-        stage_cfg.insert(prefill_config.begin(), prefill_config.end());
-    } else if (config_name == "GENERATE_CONFIG") {
-        std::map<std::string, std::string> generate_config = {
-            { "NPU_USE_NPUW", "YES" },
-            { "NPUW_FOLD", "YES" },
-            { "NPUW_DCOFF_TYPE", "f16" },
-            { "NPUW_DCOFF_SCALE", "YES" },
-            { "NPU_COMPILATION_MODE_PARAMS", "compute-layers-with-higher-precision=Sqrt,Power,ReduceMean,Add" },
-            { "NPUW_PARALLEL_COMPILE", "YES" },
-            { "NPUW_FUNCALL_ASYNC", "YES" },
-            { "NPUW_ONLINE_AVOID", "P:RMSNorm/NPU" }
-        };
-        stage_cfg.insert(generate_config.begin(), generate_config.end());
+void merge_config_with(ov::AnyMap& rhs, const ov::AnyMap& lhs) {
```

Comment:
your rhs comes first and lhs second, is it expected?

Reply:
No, thanks for catching this! Fixed